### DISCUSSION TITLE
Description of the repo

### DISCUSSION THREAD

**Discussion by goto40** [tcbrindle/flux#226] [Q&A]
Is "sequences-orientated" correct? Should it maybe changed to "collection-orientated"?

I would be happy to clarify this in this discussion.

I am motivated by these references:
* [Collection Oriented Programming](https://wiki.c2.com/?CollectionOrientedProgramming)
* [Parallel Block-Delayed Sequences](https://dl.acm.org/doi/pdf/10.1145/3503221.3508434)

**Comment by tcbrindle** [comment]
I've never really seen a formal description of either term, so I've been using "sequence-orientated" and "collection-orientated" basically as synonyms. I'm happy to switch to using one or the other if there's established precedent.

> I would be happy to clarify this in this discussion.

Please feel free :)
